xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -754,20 +728,21 @@ struct UserProfileDetailModel: ModelProtocol {
                 UserProfileDetailAction.succeedFollow(petname)
             })
             .recover { error in
-                switch error {
-                case AddressBookError.invalidAttemptToOverwitePetname:
-                    return .failFollowDueToPetnameCollision(
-                        petname: petname
-                    )
-                case _:
-                    return .failFollow(
-                        error: error.localizedDescription
-                    )
-                }
+                .failFollow(
+                    error: error.localizedDescription
```

Comment:
If we're presenting the `localizedDescription` to users, we may want to do a pass on the errors that could happen here and make them user-friendly.

Reply:
Currently we're swallowing them and presenting generic errors instead (see `UserProfileDetailAction.toAppAction`).

I don't think there's much benefit in surfacing Noosphere errors to the user?